### ISSUE TITLE
"bad box: x1 larger than x2"

### ISSUE THREAD

**Issue by x-x110** [chensnathan/YOLOF#8]
In the course of my training, I made this error but it was predicting delta。in the logic, it shouldn't have occurred , whether or not I downloaded the trained weights, I would have thrown this error when I iterated around 400。

**Comment by chensnathan** [comment]
Hi,
Could you provide more details about how you train with YOLOF?

**Comment by x-x110** [comment]
I'll download the code, as you said.  Since your code uses distributed training by default, and I only have one graphics card, I annotated the YoloF.py   (num_foreground) and num_foreground = num_foreground * 1.0 / dis.get_world_size ()    inside. Start training on this basis.

Note: I use your weights to be able to test.

**Comment by chensnathan** [comment]
Please check [here](https://github.com/chensnathan/YOLOF/issues/3#issuecomment-805452062) to adjust the learning rate and batch size with one GPU.

**Comment by x-x110** [comment]
think you!!!!!
